### DISCUSSION TITLE
Cadaver - custom property

### DISCUSSION THREAD

**Discussion by GitBobAye** [notroj/cadaver#30] [Q&A] [answered]
I have a file and am trying to use cadaver to set the executable property. I can see the property is currently set to false:

http://apache.org/dav/props/ executable = F

I can't figure-out the syntax to set this to true. I've tried the following:

propset test.sh http://apache.org/dav/props/:executable true
propset test.sh "http://apache.org/dav/props/ executable" true

On the server side I get the following error:

AH00541: XML Parser Error: XML parser error code: not well-formed (invalid token) (4)

Any suggestions?

**Comment by notroj** [comment, marked as answer]
The namespace used for propset/propget is an option, so can you try:

```
  set namespace http://apache.org/dav/props/
  propset test.sh executable T

```
mod_dav uses **T** or **F** for the executable "bit".

**Comment by GitBobAye** [comment]
This worked as I had hoped. Thank you.
